### DISCUSSION TITLE
앱 스크롤 끝 효과

### DISCUSSION THREAD

**Discussion by kang9366** [devocean-green-dev/GreenDev_BE#6] [Polls]
앱 스크롤 끝 효과가 전체적인 앱 디자인과 어울리지 않는 것 같은데 변경하는 것이 좋을지 그대로 두는 것이 좋을지 의견 부탁드릴게요!

<img src="https://github.com/devocean-green-dev/GreenDev_BE/assets/63611804/1b295246-691a-4ba0-be9d-30d6d6adaef2" width="30%" height="30%"/>

**Comment by yymin1022** [comment]
요게아마 API버전이 높은 기기에서는 색상대신 늘어나는 형태로애니메이션이 나올거에요!
연한 초록색 계열로 처리하면 괜찮을까 싶네요,,
Theme에 색상 속성들 자체를 초록계열로 처리하면 어떨까요? `colorPrimary`, `colorAccent` 등등..

**Reply by kang9366** [reply]
넵 알겠습니다~!
